app/jobs/invoices/prepaid_credit_job.rb

```diff
@@ -4,6 +4,7 @@ module Invoices
   class PrepaidCreditJob < ApplicationJob
     queue_as 'wallets'
 
+    retry_on ActiveRecord::StaleObjectError, wait: :polynomially_longer, attempts: 6
```

Comment:
would you consider adding a test case to ensure the jobs are retried when the job fails with this kind of error?